libs/eslint-plugin-vx/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import rules from './rules'
+import all from './configs/all'
+
+export = {
+  rules,
+  configs: {
+    all,
+  },
+}
```

Comment:
This whole package was pretty heavily cargo-culted from `@typescript-eslint/eslint-plugin`.